rosdep/base.yaml

```
@@ -7216,6 +7216,18 @@ muparser:
   nixos: [muparser]
   opensuse: [muparser-devel]
   ubuntu: [libmuparser-dev]
+nanobind-dev:
+  arch:
+    pacman:
+      packages: [nanobind]
+  debian: [nanobind-dev]
+  freebsd: [nanobind]
+  ubuntu:
+    noble: [nanobind-dev]
+    oracular: [nanobind-dev]
+    "*":
+      pip:
+        packages: [nanobind]
```

We should do this the other way around so that this will work into the future:

```suggestion
  ubuntu:
    '*': [nanobind-dev]
    focal:
      pip:
        packages: [nanobind]
    jammy:
      pip:
        packages: [nanobind]
```